packages/interactive_media_ads/lib/src/platform_interface/interactive_media_ads_platform.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:plugin_platform_interface/plugin_platform_interface.dart';
+
+import 'platform_ad_display_container.dart';
+import 'platform_ads_loader.dart';
+import 'platform_ads_manager_delegate.dart';
+
+/// Interface for a platform implementation of the Interactive Media Ads SDKs.
+abstract class InteractiveMediaAdsPlatform extends PlatformInterface {
```

Comment:
Since this is new, maybe we should just use `base`, per https://github.com/flutter/flutter/issues/127396

Reply:
I reimplemented `InteractiveMediaAdsPlatform` and the platform interface classes with base. So I could no longer use `mockito` and I had to write a stub for all the classes. See `test/test_stubs.dart`. Writing stubs for everything takes more time than using mockito, but now we no longer have to test "can't implements" and "can extend" for the platform interface classes. And I think adding new methods to the stubs should be quick moving forward.